src/react-components/preferences-screen.js

```diff
@@ -509,8 +517,20 @@ class PreferenceListItem extends Component {
     const intl = this.props.intl;
     const isCheckbox = this.props.itemProps.prefType === PREFERENCE_LIST_ITEM_TYPE.CHECK_BOX;
     const isSmallScreen = window.innerWidth < 600;
+    const onHelpIconClick = () => {
+      window.open(preferenceLabels[this.props.storeKey].helpLink, "_blank");
+    };
     const label = (
-      <span className={styles.preferenceLabel}>{intl.formatMessage(preferenceLabels[this.props.storeKey])}</span>
+      <span className={styles.preferenceLabel}>
+        {intl.formatMessage(preferenceLabels[this.props.storeKey])}
+        {preferenceLabels[this.props.storeKey].helpLink !== undefined ? (
+          <IconButton onClick={onHelpIconClick}>
+            <HelpIcon />
+          </IconButton>
+        ) : (
+          <></>
+        )}
+      </span>
```

Comment:
![image](https://user-images.githubusercontent.com/7637832/114447394-38c11400-9b87-11eb-8d14-e175883c1d37.png)

I wanted to put the help icon in the same line of labels but I couldn't avoid the new line. I guess `intl.formatMessage()` adds a new line? Is it possible to remove the new line?

Reply:
I would suggest something this, where you use React's `as` property to turn this element into a link, instead of using javascript to open a new window, and also a class to style the link properly so that it appears on the same line:

```javascript
<IconButton
  className={styles.helpButton}
  as="a"
  href={preferenceLabels[this.props.storeKey].helpLink}
  target="_blank"
>
```

```scss
/* preferences-screen.scss */
:local(.help-button) {
  display: inline;

  svg {
    display: inline;
    vertical-align: text-bottom;
    margin-left: 5px;
  }
}
```

Reply:
Updated, thanks!